### DISCUSSION TITLE
Vulnerabilities not showing CVEs

### DISCUSSION THREAD

**Discussion by DerGast01** [wazuh/wazuh-kibana-app#5495] [Q&A]
Hi at all!
I'm using Wazuh (the latest version, on Ubuntu) for a Windows Environment.
I've activated the Vulnarabilities Scan. It works fine for some Clients and Servers, but not for all! :)
I reinstalled the Wazuh Agent (4.4.2.1), but there is no input, although there is a "Last Scan time" showing.
The ossec Conf. on the Agent is the same like the other Agents. 

![grafik](https://github.com/wazuh/wazuh-kibana-app/assets/134588318/92e1a91a-48d9-40eb-aca3-b8b1acee47dd)

![grafik](https://github.com/wazuh/wazuh-kibana-app/assets/134588318/29a60a26-7d4c-4785-8ace-c8adc7d88b27)

Any ideas?

Kind regards 
Daniel

**Comment by Tostti** [comment]
Hello @DerGast01, hope you are well!

I will be helping you to solve this issue, I have a few questions: 

- Which version of Windows have the agents that are failing?
- Are you applying syscollector configuration via Centralized configuration? 

The log `Unable to send file 'merged.mg' to agent ID '020'` indicates that there was a problem in the communication between the server and the agent to send the file. Part of that file contains the configuration applied via Centralized configuration, meaning that if Syscollector was configured there, it is possible that the agent didn't receive the information. 
To verify that, you can navigate to `<installation path>/shared/agent.conf` and verify if its content is the expected. 

I will be looking forward to your answer! 

Kind regards,

**Reply by DerGast01** [reply]
Hey Tostti!
Thank you for your reply.
I posted my problem also in this group.
https://github.com/wazuh/wazuh/discussions/17294
Maybe there are some helpful Information for you/me?

**Comment by Tostti** [comment]
Hi @DerGast01, I see that a coworker was already providing support in https://github.com/wazuh/wazuh/discussions/17294. 
In that case, I will proceed to close this issue and refer it in the other thread so all the information is in the same place. 
Have a great day!
